ICRC-31: extract icrc31_list_identities (#103)
dfinity/wg-identity-authentication: topics/icrc_3x_get_delegation.md

# ICRC-3x: Get Delegation (ICRC-25 Extension)

[![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)

<!-- TOC -->

> - [ICRC-32: Get Delegation](#icrc-3x-get-delegation)

Reviewer: ```suggestion
- [ICRC-3x: Get Delegation](#icrc-3x-get-delegation)
```

  - [Summary](#summary)
  - [Method](#method)

>   - [Request](#request-params)

Reviewer: Request & Response are not valid references to existing headings.

  - [Response](#request-params)

## Summary

The purpose of the `icrc3x_get_delegation` method is for the relying party to receive delegation identity.

## Method

**Name and Scope:** `icrc3x_get_delegation`

> **Prerequisite:** Active session with granted permission for method name/scope.

Reviewer: ```suggestion
**Prerequisite:** Active session with granted permission scope `icrc3x_get_delegation` or `*`.
```

> # ICRC-3x: Get Delegation

Reviewer: I think we agreed to move this file to a different PR?

Author: I'm okay with both. Keeping it in this one is easier as we don't lose functionality. And as it is declared as a draft, it doesn't hurt to keep it here. What do you think?

Reviewer: Ok, fair enough. 👍
